### ISSUE TITLE
Fix compilation w/ XcodeXbeta

### ISSUE THREAD

**Issue by helje5** [SwiftXcode/swift-xcode#9]
```
Compile CNIOSHA1 c_nio_sha1.c
Compile CNIODarwin shim.c
Compile CNIOLinux shim.c
Compile CNIOZlib empty.c
error: unknown target triple 'unknown-apple-macosx10.13.0', please use -triple or -arch
error: unknown target triple 'unknown-apple-macosx10.13.0', please use -triple or -arch
error: unknown target triple 'unknown-apple-macosx10.13.0', please use -triple or -arch
error: unknown target triple 'unknown-apple-macosx10.13.0', please use -triple or -arch
error: terminated(1): /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-build-tool -f /Users/helge/tmp/HelloWOrld2/HelloWOrld2/.buildzz/.build/debug.yaml main output:
```

**Comment by helje5** [comment]
Xcode issues to the script (and itself uses!):
```
export CURRENT_ARCH=undefined_arch
export arch=undefined_arch
```
with
```
clang -v 2>&1 | grep Target
Target: x86_64-apple-darwin17.5.0
```

**Comment by helje5** [comment]
Fixed in swift-code 0.7.0
